buku.py

```diff
@@ -2611,7 +2612,12 @@ def is_bad_url(url):
     """
 
     # Get the netloc token
-    netloc = parse_url(url).netloc
+    try:
+        netloc = parse_url(url).netloc
+    except LocationParseError as e:
```

Comment:
Why not catch generic `Exception` here and print e? e will show what type it is.

Reply:
mostly to avoid pylint check. any future error may need different method.

Reply:
:+1: